x/slashing/types.go

```diff
@@ -8,3 +8,10 @@ type SideDowntimeSlashPackage struct {
 	SideChainId   types.ChainID `json:"side_chain_id"`
 	SideTimestamp uint64        `json:"side_timestamp"`
 }
+
+type SideMalicousVoteSlashPackage struct {
+	SideVoteAddr  []byte        `json:"side_vote_addr"`
```

Comment:
will `[32]byte` better than `[]byte`?

Reply:
in the sdk, use []byte to express SideVoteAddr everywhere
for consistency, I also use  []byte, but [48]byte is better indeed.